iron-autogrow-textarea.html

```diff
@@ -202,6 +202,7 @@
 
     /**
      * Returns the underlying textarea.
+     * @type HTMLTextAreaElement
```

Comment:
I think what we've been using is something like `@return {HTMLTextAreaElement} the underlying textarea`. `@type` has generally been used for properties